packages/ariakit/src/combobox/__examples__/combobox-popover-opening-with-at-least-1-character/index.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Combobox,
+  ComboboxItem,
+  ComboboxPopover,
+  useComboboxState,
+} from "ariakit/combobox";
+import "./style.css";
+
+export default function Example({ defaultValue }: { defaultValue?: string }) {
+  const combobox = useComboboxState({
+    gutter: 8,
+    sameWidth: true,
+    defaultValue,
+  });
```

Comment:
I've thought about the possibility of adding props to the root example components and showing a sort of panel on the UI so users could configure the initial props and test different behaviors. But we're not quite there, and I still don't know if it's a good idea.

For now, I'd just remove this prop and use the `type` function from `ariakit-test` on the test.

Reply:
Done.

I've added on both tests, the triplet:

```typescript
  await press.Tab();
  await type("a");
  await press.Escape();
```